airflow/models/dagbag.py

```diff
@@ -83,7 +82,6 @@ class DagBag(BaseDagBag, LoggingMixin):
     CYCLE_IN_PROGRESS = 1
     CYCLE_DONE = 2
     DAGBAG_IMPORT_TIMEOUT = conf.getint('core', 'DAGBAG_IMPORT_TIMEOUT')
-    UNIT_TEST_MODE = conf.getboolean('core', 'UNIT_TEST_MODE')
     SCHEDULER_ZOMBIE_TASK_THRESHOLD = conf.getint('scheduler', 'scheduler_zombie_task_threshold')
```

Comment:
Is this still needed in here? Feels like it shouldn't be

Reply:
I don't know. I just do simple refactoring. I don't know the history of this option.